src/sequence-steps.js

```diff
@@ -233,6 +325,13 @@ export function gitPushUpstreamMaster( [ git, options ] ) {
 		.then( data => utils.log.end() );
 }
 
+export function gitPushUpstreamFeatureBranch( [ git, options ] ) {
+	const command = `git push upstream ${ options.branch } --tags`;
+	utils.log.begin( command );
+	return utils.exec( command )
+		.then( data => utils.log.end() );
```

Comment:
i can see this failing if the user does not have write access to the upstream repo.
could you please add an advise entry for this?